docs/hub/model-cards-user-studies.md

```diff
@@ -27,17 +26,21 @@ Our user studies provided further clarity on the sections that different user pr
 
 The results illustrated below show that while the Bias, Risks and Limitations section ranks second for both model card writers and model card readers for *In what order do you write the model card and What section do you look at first*, respectively, it is also noted as the most challenging/longest section to write. This favoured/endorsed the need to further evaluate the Bias, Risks and Limitations sections in order to assist with writing this decisive/imperative section.
 
-These templates were then used to generate model cards for the top 200 most downloaded Hugging Face models. 
+These templates were then used to generate model cards for the top 200 most downloaded Hugging Face (HF) models. 
 
-* We first began by pulling all Hugging Face model's on the hub and, in particular, the Limitations and Bias sections.
+* We first began by pulling all Hugging Face model's on the hub and, in particular, subsections on Limitations and Bias ("Risks" subsections were largely not present).
```

Comment:
Is this true? I added it because the Risks weren't mentioned, and I vaguely feel like it's true from what I've read, but not sure.